ycmd/completers/general/tests/filename_completer_test.py

```diff
@@ -34,6 +34,17 @@
   'file_data' : { PATH_TO_TEST_FILE : { 'filetypes' : [ 'cpp' ] } }
 }
 
```

Comment:
Two blank lines are expected before and after functions and classes. You can check this in other files. I think this is to follow the same guideline used in the C++ code. This is also part of the [official style guide](https://www.python.org/dev/peps/pep-0008/#blank-lines) (but we don't respect all of it).